lib/Jsrt/Jsrt.cpp

```diff
@@ -1244,7 +1244,7 @@ CHAKRA_API JsGetGlobalObject(_Out_ JsValueRef *globalObject)
 
 CHAKRA_API JsCreateObject(_Out_ JsValueRef *object)
 {
-    return ContextAPINoScriptWrapper([&](Js::ScriptContext *scriptContext, TTDRecorder& _actionEntryPopper) -> JsErrorCode {
+    return ContextAPIWrapper<true>([&](Js::ScriptContext *scriptContext, TTDRecorder& _actionEntryPopper) -> JsErrorCode {
```

Comment:
@MSLaguana  is right. `JsCreateObject()` won't execute the user script and hence we want to keep it under `ContextAPINoScriptWrapper`. Additionally `ContextAPINoScriptWrapper` is lighter version of `ContextApiWrapper` as it doesn't have `BEGIN_ENTER_SCRIPT(scriptContext, true, true, true)` and `END_ENTER_SCRIPT`.